src/main/groovy/com/yodle/vantage/GenerateVantageJsonTask.groovy

```diff
@@ -34,6 +34,15 @@ class GenerateVantageJsonTask extends DefaultTask implements ReportProvider
 {
   public static final String DEFAULT_PROFILE = "default"
   public static final String LATEST_VERSION = "latest"
+  def doNotResolveConfigurations =
```

Comment:
also in:
https://docs.gradle.org/3.4/release-notes.html
"Configurations can be unresolvable"
I could not use their approach of using isCanBeResolved to determine which configuration should be removed from here, so had to populate it manually, due to older version of gradle used not having this method.